Patches/Kit's VFE Weapons/Ammo/Ammo_KitPlasma.xml

```diff
@@ -0,0 +1,150 @@
+						<label>Plasma Pistol Power Cell</label>
+						<ammoTypes>
+							<Ammo_PlasmaCellPistol>Bullet_KIT_PlasmaSMG</Ammo_PlasmaCellPistol>
+						</ammoTypes>
+					</CombatExtended.AmmoSetDef>
+					
+					<CombatExtended.AmmoSetDef>
+						<defName>AmmoSet_KIT_PlasmaRifle</defName>
+						<label>Plasma Rifle Power Cell</label>
+						<ammoTypes>
+							<Ammo_PlasmaCellRifle>Bullet_KIT_PlasmaRifle</Ammo_PlasmaCellRifle>
+						</ammoTypes>
+					</CombatExtended.AmmoSetDef>
+					
+					<CombatExtended.AmmoSetDef>
+						<defName>AmmoSet_KIT_PlasmaRifle</defName>
+						<label>Plasma Rifle Power Cell</label>
+						<ammoTypes>
+							<Ammo_PlasmaCellRifle>Bullet_KIT_PlasmaSniper</Ammo_PlasmaCellRifle>
+						</ammoTypes>
+					</CombatExtended.AmmoSetDef>
+
+					<!-- ==================== Ammo ========================== -->
+
+					<!-- ================== Projectiles ================== -->
+
+						<ThingDef Class="CombatExtended.AmmoDef" ParentName="BaseVFEIPlasmaBullet">
+						  <defName>Bullet_KIT_PlasmaPistol</defName>
+						  <label>plasma pistol shot</label>
+						  <graphicData>
+							<texPath>Things/Item/Weapons/Plasma_Beam</texPath>
+							<graphicClass>Graphic_Single</graphicClass>
+						  </graphicData>
+						  <projectile Class="CombatExtended.ProjectilePropertiesCE">
+							<damageAmountBase>15</damageAmountBase>
+							<armorPenetrationSharp>10</armorPenetrationSharp>
+							<armorPenetrationBlunt>10</armorPenetrationBlunt>
+							<empShieldBreakChance>0.2</empShieldBreakChance>
+							<secondaryDamage>
+							  <li>
+								<def>EMP</def>
+								<amount>3</amount>
+							  </li>
+							</secondaryDamage>
+						  </projectile>
+						</ThingDef>
+						
+						<ThingDef Class="CombatExtended.AmmoDef" ParentName="BaseVFEIPlasmaBullet">
+						  <defName>Bullet_KIT_PlasmaRifle</defName>
+						  <label>plasma rifle shot</label>
+						  <graphicData>
+							<texPath>Things/Item/Weapons/Plasma_Nail</texPath>
+							<graphicClass>Graphic_Single</graphicClass>
+						  </graphicData>
+						  <projectile Class="CombatExtended.ProjectilePropertiesCE">
+							<damageAmountBase>30</damageAmountBase>
+							<armorPenetrationSharp>10</armorPenetrationSharp>
+							<armorPenetrationBlunt>10</armorPenetrationBlunt>
+							<empShieldBreakChance>0.5</empShieldBreakChance>
+							<secondaryDamage>
+							  <li>
+								<def>EMP</def>
+								<amount>9</amount>
+							  </li>
+							</secondaryDamage>
+						  </projectile>
+						</ThingDef>
+						
+						<ThingDef Class="CombatExtended.AmmoDef" ParentName="BaseVFEIPlasmaBullet">
+						  <defName>Bullet_KIT_PlasmaSMG</defName>
+						  <label>plasma pistol shot</label>
+						  <graphicData>
+							<texPath>Things/Item/Weapons/Plasma_Beam</texPath>
+							<graphicClass>Graphic_Single</graphicClass>
+						  </graphicData>
+						  <projectile Class="CombatExtended.ProjectilePropertiesCE">
+							<damageAmountBase>5</damageAmountBase>

```

Comment:
Against a pawn in just plain clothing with minimal heat protection, it takes 6 - 8 rounds to down them. It seems... acceptable, to me, but you might consider whether or not you feel that's enough stopping power.

Reply:
Boosted the damage to 6. Thing to keep in mind is that it'll down most targets with the same amount of rounds, since heat protection is very hard to get a lot of. I was expecting the EMP to do like, 0.5 damage per hit or something even against organics but that doesn't seem to be the case.